x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -182,6 +185,94 @@ export const ModelDeploymentInProgress = ({
 export const ModelDeployed = ({
   dismiss,
   isDismissable,
+  isStartButtonDisabled,
+}: Pick<TextExpansionCallOutState, 'dismiss' | 'isDismissable' | 'isStartButtonDisabled'>) => {
+  const { startTextExpansionModel } = useActions(TextExpansionCalloutLogic);
+
+  return (
+    <EuiPanel color="success">
```

Comment:
As @julianrosado noted, this should be an `EuiCallOut`. That'll help with styling and keeping everything consistent across the codebase. Callout style may get updated on the EUI side while the panel may not (or the other way around) leading to wonky behavior.

Reply:
I switched to `EuiCallOut` but see [my comment about it](https://github.com/elastic/kibana/pull/156080#discussion_r1180546850).